### PR TITLE
fix(deps): switch to bignum, add Node 4 testing

### PR DESCRIPTION
**Pull request by vladikoff**

### CODE REVIEW

File: libs/bigint-patch.js [hunk 1]
```diff
@@ -79,7 +79,7 @@ try {
       return this._bigint.cmp(other._bigint);
     },
     toString: function(base) {
-      return this._bigint.toString(base || 10);
+      return this._bigint.toString(base || 10).replace(/^0/, '');
```
Comment (rfk) [inline libs/bigint-patch.js]: Will there only ever be at most one leading zero?

**Comment by vladikoff** [comment]
RE: performance (as it was mentioned before)

I did some measurements for perf in actual tests and not the overall test runner and the changes performed better  node 4: ` OK » 100 honored (0.813s) ` vs  old version with node 0.10`✓ OK » 100 honored (3.440s) `

**Comment by rfk** [comment]
LGTM

File: libs/bigint-patch.js (hunk 1 above)
Comment (vladikoff) [inline libs/bigint-patch.js]: Should be yea!

**Pull request merged**